mne/export/_edf.py

```diff
@@ -309,24 +324,15 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
                 raw.annotations.duration,
                 raw.annotations.ch_names,
             ):
-                # annotations are written in terms of 100 microseconds
-                onset = onset * 10000
-                duration = duration * 10000
+                # annotations are written in terms of seconds
                 if ch_names:
                     for ch_name in ch_names:
-                        if (
+                        try:
                             hdl.writeAnnotation(onset, duration, desc + f"@@{ch_name}")
-                            != 0
-                        ):
+                        except Exception:
```

Comment:
```suggestion
                        except Exception as e:
```
This way, we can access `e` later. See comment below.